android/src/main/java/com/swmansion/reanimated/NodesManager.java

```diff
@@ -78,6 +78,7 @@
   private List<OnAnimationFrame> mFrameCallbacks = new ArrayList<>();
   private ConcurrentLinkedQueue<Event> mEventQueue = new ConcurrentLinkedQueue<>();
   private boolean mWantRunUpdates;
+  private boolean preventOnAnimationFrame = false;
```

Comment:
If we check this in applier we do we need it here?

Reply:
It is needed here. Checking from Applier has been moved to NativeProxy on iOS, but on Android moving it to NativeProxy does not work as expected

Reply:
933088b